manim/mobject/geometry.py

```diff
@@ -82,13 +88,17 @@ def construct(self):
 from ..utils.space_ops import (
     angle_between_vectors,
     angle_of_vector,
-    compass_directions,
     line_intersection,
     normalize,
     regular_vertices,
     rotate_vector,
 )
 
+if TYPE_CHECKING:
+    from numpy import float64, ndarray
```

Comment:
in the whole codebase, we have a convention to import `numpy` as `np`. Please change this.

Reply:
Leftover from MonkeyType. Guess I'll have to remember to fix that. Wait... I'm not importing NumPy just stuff from NumPy, so no np.

Reply:
you shouldn't do so. everywhere else `numpy` is imported as `np` and the things imported should be accessed as `np.<things>`. The same should be followed here.

Reply:
But I don't need to import numpy, so why should I?

Reply:
I mean you have to change everything using `float64` and `ndarray` to `np.float64` and `np.ndarray`.